doc/reference/reference_lua/box_events/system_events.rst

```diff
@@ -99,6 +101,26 @@ Contains schema-related data.
     MP_STR “version”: MP_UINT schema_version,
     }
 
+.. _system-events_box-shutdown:
+
+box.shutdown
+~~~~~~~~~~~~
+
+Contains a boolean value which states if there is an active shutdown request.
+
+``box.shutdown`` is supposed to be used with connectors to implement the graceful shutdown protocol.
+
+The event is generated when the server receives a shutdown request (``os.exit()`` command or
+:ref:`SIGTERM <admin-server_signals>` signal).
+The server calls :ref:`box.broadcast('box.shutdown', true) <box-broadcast>`
+from the :ref:`box.ctl.on_shutdown() <box_ctl-on_shutdown>` trigger callback.
+Then the server stops accepting new connections.
+It waits for all the subscribed connections to be closed.
+The client receives the event marked as ``true`` and closes the connection gracefully.
+If all the subscribed connections are closed, the server will be shutdown.
+Otherwise, a timeout occurs.
```

Comment:
"A timeout occurs" and what?

Generally, I find this text hard to read. I can't put my finger on it, but it looks... disconnected, mechanical. Is it possible to make it an easy read? @patiencedaur